index.html

```diff
@@ -6,10 +6,136 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Web pz 1.4</title>
+  <link rel="stylesheet" href="css/style.css">
+  <link rel="stylesheet" href="css/header.css">
+  <link rel="stylesheet" href="css/main.css">
+  <link rel="stylesheet" href="css/sidebar.css">
 </head>
 <body>
-  <div>
+<!--    header   -->
+<header class="header">
+  <img id="open-sidebar" class="header-icon" src="img/menu.svg" alt="menu">
```

Comment:
Please use relative path for all src or href

Reply:
Ok